src/App.jsx

```diff
@@ -1,35 +1,38 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import Articles from "./components/Articles";
+import Header from "./components/Header";
+import Navbar from "./components/Navbar";
+import "bootstrap/dist/css/bootstrap.min.css";
+import PageNavigation from "./components/PageNavigation";
+import { useState } from "react";
+import ItemsPerPage from "./components/ItemsPerPage";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [pageNumber, setPageNumber] = useState(1);
```

Comment:
You seem to have a lot of global state that you're having to pass around - would making use of React Context be a neater solution?